src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -828,6 +836,72 @@ private void ShowTemplateHelp()
             ParameterHelp(allParams, additionalInfo, _hostSpecificTemplateData.HiddenParameterNames);
         }
 
+        private void ShowTemplateNameMismatchHelp()
+        {
+            if (_matchedTemplates.Count == 0)
+            {
+                Reporter.Error.WriteLine("No installed template's names were similiar enough.");
```

Comment:
This message got removed, as it's effectively a duplicate of the later message indicating there were no matches, i.e.

> Reporter.Error.WriteLine($"No templates matched the input template name: [{_templateName}]");